graylog2-server/src/main/java/org/graylog2/plugin/lookup/LookupDataAdapter.java

```diff
@@ -237,6 +237,14 @@ public LookupResult assignTtl(Object key, Long ttlSec) {
         return resultWithError;
     }
 
+    public boolean supportsPreview() {
```

Comment:
Noticed this isn't used anywhere, not sure if it was added as a just a future proofing measure but just a heads up

Reply:
I am assuming we will also add the same feature to Data Adapters so I just went ahead and added it here to use when we do